### PR TITLE
Domain admin tests

### PR DESCRIPTION
**Pull request by JanuszJakubiec**
Adding example tests to graphql_stats_SUITE and tests to graphql_roster_SUITE. This PR shows how to add tests when the domain is taken from the domain variable and user variable.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/132097) / small_tests / e918f4304ec63bb32a89039860659b67fa12a498
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3730/132097/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3730/132097/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-08-11_12.07.11/lib.mongooseim.logs/run.2022-08-11_12.07.12/suite.log.html)

File: big_tests/tests/graphql_roster_SUITE.erl [hunk 1]
```diff
@@ -486,6 +499,51 @@ user_get_nonexistent_contact_story(Config, Alice) ->
...
+
+domain_admin_subscribe_all_to_all_story(Config, Alice, Bob, Kate) ->
+    Res = domain_admin_subscribe_all_to_all(make_contacts([Alice, Bob, Kate]), Config),
+    check_if_created_succ(?SUBSCRIBE_ALL_TO_ALL_PATH, Res),
+
+    check_contacts([Bob, Kate], Alice),
+    check_contacts([Alice, Kate], Bob),
+    check_contacts([Alice, Bob], Kate).
+
+
```
Comment (JanuszJakubiec) [inline big_tests/tests/graphql_roster_SUITE.erl:546]:
```suggestion
```

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/132201) / small_tests / 53b67a18a8a35f56334b0083de0aa0dd810f70a6
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3730/132201/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3730/132201/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-08-11_13.29.41/lib.mongooseim.logs/run.2022-08-11_13.29.41/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/132222) / small_tests / 3ec35269b5982a400200f4dd138bdee8bd540a17
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3730/132222/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3730/132222/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-08-11_13.32.48/lib.mongooseim.logs/run.2022-08-11_13.32.48/suite.log.html)

File: big_tests/tests/graphql_helper.erl [hunk 2]
```diff
@@ -84,10 +90,15 @@ init_user(Config) ->
 
 init_domain_admin_handler(Config) ->
     Domain = domain_helper:domain(),
-    Password = base16:encode(crypto:strong_rand_bytes(8)),
-    Creds = {<<"admin@", Domain/binary>>, Password},
-    ok = domain_helper:set_domain_password(mim(), Domain, Password),
-    add_specs([{domain_admin, Creds}, {schema_endpoint, domain_admin} | Config]).
+    case (not ct_helper:is_ct_running())
```
Comment (chrzaszcz) [inline big_tests/tests/graphql_helper.erl:93]: I think that in big tests we always expect CT to be running. Anyway, this condition does not make sense to me, as it does not mean that we don't have to check the DB.

File: big_tests/tests/graphql_helper.erl [hunk 3]
```diff
@@ -29,10 +29,16 @@ execute_command(Category, Command, Args, Config) ->
...
     execute_command(Category, Command, Args, Config, Protocol).
 
+execute_domain_admin_command(Category, Command, Args, Config) ->
+    execute_command(Category, Command, Args, Config, domain).
+
 %% Admin commands can be executed as GraphQL over HTTP or with CLI (mongooseimctl)
 execute_command(Category, Command, Args, Config, http) ->
     #{Category := #{Command := #{doc := Doc}}} = get_specs(),
     execute_auth(#{query => Doc, variables => Args}, Config);
+execute_command(Category, Command, Args, Config, domain) ->
```
Comment (chrzaszcz) [inline big_tests/tests/graphql_helper.erl:39]: Please don't use `domain` for Protocol. The protocol is still HTTP. Just put lines 40-41 directly in the body of `execute_domain_admin_command/4`

File: big_tests/tests/graphql_roster_SUITE.erl [hunk 4]
```diff
@@ -486,6 +498,51 @@ user_get_nonexistent_contact_story(Config, Alice) ->
...
```
Comment (chrzaszcz) [inline big_tests/tests/graphql_roster_SUITE.erl:545]: Minor: double newline

**Review by chrzaszcz — COMMENTED** [review]
I think we could check a real domain instead of a non-existing one. How about using `ct:get_config({hosts, mim, secondary_domain})` - btw I think that we could have a helper for it in `domain_helper`:

```
secondary_domain() ->
    get_or_fail({hosts, mim, secondary_domain}).
```

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/132523) / small_tests / f3b74ef9f61e50c1d787868cad13c09c1f9dc489
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3730/132523/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3730/132523/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-08-12_12.44.48/lib.mongooseim.logs/run.2022-08-12_12.44.48/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks good 👍

**Pull request merged**